erizo/src/erizo/MediaStream.h

```diff
@@ -170,6 +174,8 @@ class MediaStream: public MediaSink, public MediaSource, public FeedbackSink,
   bool video_muted_;
 
   bool pipeline_initialized_;
+
+  bool is_publisher_;
```

Comment:
I prefer having a boolean since we don't need more functionality yet. We can change it in the future anyway. But I'll add the info to the `toLog()` inline.